lib/takeaway.rb

```diff
@@ -0,0 +1,47 @@
+class Takeaway
+  attr_reader :basket
+
+  def initialize 
+    @menu = {"pasta": 7, "dhal": 9 , "pizza": 10 , "chocolate": 9} 
+    @basket = { }
+    @total_cost = 0
+    @basket_with_cost = {}
+  end
+
+  def view_menu
+    @menu
```

Comment:
Potentially could do a .each
So that it doesn't just show the actual hash. To show the items with their price in a list?

Reply:
Nice idea, @OliviaMaxwell !